connectors/google-maps-platform/element-templates/google-maps-platform-connector.json

```diff
@@ -0,0 +1,533 @@
+{
+  "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name": "Google Maps Platform Connector",
+  "id": "io.camunda.connectors.GoogleMapsPlatform.v1",
+  "description": "Orchestrate your Google Maps Platform operations with Camunda. You can validate and format address, get postal address, calculate distance, etc.",
+  "documentationRef": "https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/google-maps-platform/",
+  "category": {
+    "id": "connectors",
+    "name": "Connectors"
+  },
+  "appliesTo": [
+    "bpmn:Task"
+  ],
+  "elementType": {
+    "value": "bpmn:ServiceTask"
+  },
+  "groups": [
+    {
+      "id": "operation",
+      "label": "Operation"
+    },
+    {
+      "id": "authentication",
+      "label": "Authentication"
+    },
+    {
+      "id": "configuration",
+      "label": "Configuration"
+    },
+    {
+      "id": "input",
+      "label": "Input"
+    },
+    {
+      "id": "output",
+      "label": "Output"
+    },
+    {
+      "id": "errors",
+      "label": "Error Handling"
+    }
+  ],
+  "properties": [
+    {
+      "type": "Hidden",
+      "value": "io.camunda:http-json:1",
+      "binding": {
+        "type": "zeebe:taskDefinition:type"
+      }
+    },
+    {
+      "label": "Operation type",
+      "id": "operationType",
+      "group": "operation",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "Validate an address",
+          "value": "validateAddress"
+        },
+        {
+          "name": "Get place ID",
+          "value": "getPlaceID"
+        },
+        {
+          "name": "Directions",
```

Comment:
what do you mean with directtions?

Do you mean distance? then call it: Calculate Distance